relay-server/src/utils/rate_limits.rs

```diff
@@ -563,6 +568,8 @@ where
                 // Otherwise, the outcome is logged at a different place.
                 if !summary.transaction_metrics_extracted {
                     enforcement.event_metrics = CategoryLimit::new(category, 1, longest);
+                    enforcement.span_metrics =
+                        CategoryLimit::new(DataCategory::Span, summary.span_quantity, longest);
```

Comment:
`longest` can be none, should we set the enforcement on spans / span_metrics in that case?

Reply:
This was a question I also asked myself and given my lack of context of the product here, I don't know what we expect.

Right now the logic, in case the transaction check fails, it creates a default `span_metrics` so it will actually enter in the span rate limiting below.